schemas/test/en/test_optional_guidance_and_description.json

```diff
@@ -0,0 +1,216 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Other input fields",
+    "theme": "default",
+    "description": "A questionnaire to demo optional question guidance and descriptions",
+    "metadata": [
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        },
+        {
+            "name": "user_id",
+            "type": "string"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Linear",
+        "options": {
+            "summary": {
+                "collapsible": false
+            }
+        }
+    },
+    "sections": [
+        {
+            "id": "default-section",
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "description-block",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "answer-1",
+                                        "label": "Enter any description text that you want to be displayed",
+                                        "max_length": 20,
+                                        "mandatory": false,
+                                        "type": "TextField"
+                                    },
+                                    {
+                                        "id": "answer-2",
+                                        "label": "Enter any description text that you want to be displayed",
```

Comment:
Maybe change description to `Enter any alternative text you want to be displayed` or have question guidance to mention that its first non empty item that gets shown? might just be as I hadn't seen the block before but I assumed if one of them was empty that it would make a difference, but both need to be

It was clear enough after trying though so don't mind if you just resolve this comment